src/Conversion/ONNXToKrnl/NN/Conv.cpp

```diff
@@ -81,129 +92,142 @@ struct ONNXConvOpLowering : public ConversionPattern {
     MemRefType tmpType = MemRefType::get({}, memRefType.getElementType());
     // Single scalar, no need for default alignment.
     Value reductionVal = createMemRef.alloca(tmpType);
+    // createKrnl.iterateIE(outerLoops, outerLoops, outerLbs, outerUbs,
+    //    [&](KrnlBuilder &createKrnl, ValueRange outerIndices) {
+    // createScf.parallelLoop(parLbs, parUbs, steps,
+    //    [&](DialectBuilder &createScf, ValueRange outerIndices) {
+    //      bodyFunction(outerIndices);
+    //    }
+    auto bodyFunction = [&](ValueRange outerIndices) {
```

Comment:
Did you do any changes here? Or is it just the indentation. If there are changes, will review in details